### PR TITLE
vega-2406 - Minor content changes and fixes to draft LPA form

### PR DESCRIPTION
**Pull request by mabely**
[vega-2406](https://opgtransform.atlassian.net/browse/VEGA-2406)

## Checklist

- [x] I have updated the end-to-end tests to work with my changes
- [x] I have added styling for Dark Mode
- [x] I have checked that my UI changes meet accessibility standards

### CODE REVIEW

File: internal/server/create_draft_test.go [hunk 1]
```diff
@@ -246,14 +243,11 @@ func TestPostCreateDraftWhenValidationError(t *testing.T) {
 		Return(sirius.User{Roles: []string{"private-mlpa"}}, nil)
 	client.
 		On("RefDataByCategory", mock.Anything, sirius.CountryCategory).
-		Return([]sirius.RefDataItem{{Handle: "GB", Label: "Great Britain"}}, nil)
+		Return([]sirius.RefDataItem{}, nil)
```
Comment (gregtyler) [inline internal/server/create_draft_test.go:246]: Worth keeping this to make sure the list of the countries from the API (which this is mocking) is passed to the template

File: internal/server/create_draft_test.go [hunk 2]
```diff
@@ -262,7 +256,7 @@ func TestPostCreateDraftWhenValidationError(t *testing.T) {
 	template := &mockTemplate{}
 	template.
 		On("Func", mock.Anything, createDraftData{
-			Countries: []sirius.RefDataItem{{Handle: "GB", Label: "Great Britain"}},
+			Countries: []sirius.RefDataItem{},
```
Comment (gregtyler) [inline internal/server/create_draft_test.go:259]: And similarly should keep this to perform the check

**Comment by townxelliot** [comment]
Have you auto-selected the country in the form, given that you're now not defaulting it? Or does it just get a validation error if you don't select anything?

**Comment by mabely** [comment]
> @mabely Have you auto-selected the country in the form, given that you're now not defaulting it? Or does it just get a validation error if you don't select anything?

There's a ticket for it - https://opgtransform.atlassian.net/browse/VEGA-2449 - removed it because it was making it behave weirdly with the new validation. It didn't seem straight forward to set a default value in the template

**Review by townxelliot — APPROVED** [review]

**Pull request merged**